### DISCUSSION TITLE
我为独立开发者开发落地页模板（下） | J实验室

### DISCUSSION THREAD

**Discussion by giscus** [weijunext/new-blog#5] [General]
# 我为独立开发者开发落地页模板（下） | J实验室

全栈工程师，Next.js开源手艺人，掘金签约作者，折腾才有未来

https://weijunext.com/article/ship-langding-page-fast2

**Comment by keepchen** [comment]
发现一名宝藏前端，正在研究你和 [huglemon](https://github.com/huglemon/inwind-landing-page) 的落地页项目，感谢你们的贡献🙏

**Comment by mrstanley** [comment]
购买了插件开发，这个是附赠的课程么！感谢老师，又节省好多工作量
